src/test/java/org/junit/tests/assertion/AssertionTest.java

```diff
@@ -1,5 +1,14 @@
 package org.junit.tests.assertion;
 
+import org.junit.Assert;
```

Comment:
we put static imports before non-static ones. See
https://google-styleguide.googlecode.com/svn/trunk/javaguide.html

when in doubt, don't reorder imports.